app/commands/solution/generate_help_file.rb

```diff
@@ -0,0 +1,26 @@
+class Solution
+  class GenerateHelpFile
+    include Mandate
+
+    initialize_with :solution
+
+    def call
+      "# Help\n\n#{tests}\n\n#{submitting}\n\n#{help}"
```

Comment:
Could you use [HEREDOC](https://www.rubyguides.com/2018/11/ruby-heredoc/) for this rather than embedding the newlines pls?

Reply:
Just this specific example or also the others?

Reply:
The others too.